server/plugin/command.go

```diff
@@ -367,9 +416,44 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
 
-	return msg
+	post := &model.Post{
+		ChannelId: args.ChannelId,
+		UserId:    p.BotUserID,
+		Message:   msg,
+	}
+
+	if _, appErr := p.API.CreatePost(post); appErr != nil {
+		p.API.LogWarn("error while creating post", "post", post, "error", appErr.Error())
+		return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, appErr.Error())
+	}
+
+	message := fmt.Sprintf("Successfully subscribed to [%s](%s) with events: %s.", repo, repoLink, subscribeEvents.FormattedString())
```

Comment:
1) Let's also include the org there.

2) Should the repo be inside of a code block similar to the events?

Reply:
@hanzei for 1 comment sure , did it

2 nd comment i didn't got that , are you referring to have a separate type like we did for events formatting etc ?